index.js

```diff
@@ -30,7 +30,13 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+function getAllMovieTitles(movies) {
+  if (movies.length > 0) {
+     return movies.map(movie => movie.title) 
+  } else {
+    throw 'Error: Movies Array Empty';
+  }
+}
```

Comment:
https://www.loom.com/share/2dfe4d19c5234354a8c3192a6a43fcfc

Reply:
This is great thanks Tim. Love the video critique! And I agree, consilidating error handling into a dedicated error handler for all the code is definitely something I want to move towards!